extensions/src/main/java/org/pentaho/platform/web/http/api/resources/ActionResource.java

```diff
@@ -88,14 +86,14 @@ public Response invokeAction(
     @QueryParam( ActionUtil.INVOKER_ACTIONID ) String actionId,
     @QueryParam( ActionUtil.INVOKER_ACTIONCLASS ) String actionClass,
     @QueryParam( ActionUtil.INVOKER_ACTIONUSER ) String actionUser,
-    final String actionParams ) {
+    final ActionParams actionParams ) {
```

Comment:
This change will cause the ActionInvokerSystemListener to fail, as it's calling for the `ActionResource().invokeAction( String , String , String, String )`